<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastrarReceptor</class>
 <widget class="QMainWindow" name="CadastrarReceptor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>572</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>572</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>572</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255,118,118,1), stop:1 rgba(255,210,210,1));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>572</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>572</height>
    </size>
   </property>
   <widget class="QFrame" name="frCadastroReceptor">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>60</y>
      <width>871</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 50px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="txtCadastroInstituicao">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: 700;
font-size: 30px;
line-height: 36px;
color: #000000;

</string>
     </property>
     <property name="text">
      <string>Cadastrar Receptor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAdd">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>380</y>
       <width>251</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #E13833;
border-radius: 10px;
font-weight: 700;
font-size: 18px;
color: #FFFFFF;

</string>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputNome">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>521</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background: #FFFFFF;
border: 0.5px solid #000000;
border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtNome">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: 700;
font-size: 22px;
line-height: 19px;
color: #000000;
</string>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtTipoSangue">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>110</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: 700;
font-size: 22px;
line-height: 19px;
color: #000000;

</string>
     </property>
     <property name="text">
      <string>Tipo Sangu√≠nio</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboTipoSangue">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>150</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
color: black;
Border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputCPF">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background: #FFFFFF;
border: 0.5px solid #000000;
border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtCPF">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: 700;
font-size: 22px;
line-height: 19px;
color: #000000;
</string>
     </property>
     <property name="text">
      <string>CPF</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtDataNascimento">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>250</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: 700;
font-size: 22px;
line-height: 19px;
color: #000000;

</string>
     </property>
     <property name="text">
      <string>Data de Nascimento</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateNascimento">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>290</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
color: black;
font-size: 22px;
padding-left: 10px;
border-radius: 10px;</string>
     </property>
     <property name="minimumDate">
      <date>
       <year>1900</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonReturn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>40</horstretch>
      <verstretch>40</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

background: #E13833;
border-radius: 10px;
color:#ffffff;
}
QPushButton:hover{
background: #992C29;
border-radius: 10px;
color:#ffffff;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../../resouces.qrc">
      <normaloff>:/assets/icons/return.png</normaloff>:/assets/icons/return.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../../resouces.qrc"/>
 </resources>
 <connections/>
</ui>
